config/manager/manager.yaml

```diff
@@ -9,7 +9,7 @@ apiVersion: apps/v1
 kind: Deployment
 metadata:
   name: controller-manager
-  namespace: openshift-storage
+  namespace: system
```

Comment:
Shouldn't this be `openshift-storage` to match the Namespace, above?

Reply:
no the namespace here is modified/overriden by `kustomize` so it should stay `system`